### PR TITLE
Referees 2018/configurable files

### PR DESCRIPTION
**Pull request by idanstark42**

### CODE REVIEW

File: controls.html [hunk 1]
```diff
@@ -30,6 +31,25 @@
...
+                <label>Source</label>
+                <input ng-model="track.source">
+            </p>
+            <p>
+                <label>Start</label>
+                <input ng-model="track.start">
+            </p>
+        </fieldset>
+        <p>
+            <button ng-click="config.tracks.push({})">New soundtrack</button>
```
Comment (ThinkRedstone) [inline controls.html]: This button doesn't work, maybe you need to run $apply somewhere?

File: js/helpers/ng-config.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	                config: urlConfig || clockConfig
+	            });
+
+	            _config = $storage.config;
+	    	}
+	    	return _config;
+	    },
+	    setToUrl: function(config) {
+	        console.log(config);
+	        $window.history.pushState(config,'','/?state='+JSON.stringify(config));
```
Comment (ThinkRedstone) [inline js/helpers/ng-config.js]: $window is undefined. Maybe window?

File: controls.html (hunk 1 above)
Comment (idanstark42) [reply controls.html]: It works for me... It should just add a new, empty soundtrack to the list of them, can you check and tell me why didn't it work for you?

File: js/helpers/ng-config.js (hunk 2 above)
Comment (idanstark42) [reply js/helpers/ng-config.js]: Fixed it

File: controls.html (hunk 1 above)
Comment (ThinkRedstone) [reply controls.html]: It has a problem when tracks is uninitialised- it tries to push an empty object to an undefined variable. This probably means there's a problem with reading the default config. I also noticed it doesn't play any actual sound. Not plays them quietly- chromium is reporting no sound is a playing from that tab.

Comment (idanstark42) [reply controls.html]: Fixes it

Comment (ThinkRedstone) [reply controls.html]: @idanstark42 can you push the commit?

**Pull request merged**